### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by iamrudra** [sahat/hackathon-starter#1020]
```
➜  hackathon-starter nvm use node
Now using node v13.0.1 (npm v6.13.0)

➜  hackathon-starter npm install

> node-sass@4.13.0 install /Users/vnmaniyar/Documents/local-repos/hackathon-starter/node_modules/node-sass
> node scripts/install.js

Cached binary found at /Users/vnmaniyar/.npm/node-sass/4.13.0/darwin-x64-79_binding.node

> bcrypt@3.0.6 install /Users/vnmaniyar/Documents/local-repos/hackathon-starter/node_modules/bcrypt
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download
node-pre-gyp WARN Pre-built binaries not installable for bcrypt@3.0.6 and node@13.0.1 (node-v79 ABI, unknown) (falling back to source compile with node-gyp)
node-pre-gyp WARN Hit error unable to get local issuer certificate
gyp WARN install got an error, rolling back install
gyp ERR! configure error
gyp ERR! stack Error: unable to get local issuer certificate
gyp ERR! stack     at TLSSocket.onConnectSecure (_tls_wrap.js:1321:34)
gyp ERR! stack     at TLSSocket.emit (events.js:210:5)
gyp ERR! stack     at TLSSocket._finishInit (_tls_wrap.js:794:8)
gyp ERR! stack     at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:608:12)
gyp ERR! System Darwin 18.7.0
gyp ERR! command "/Users/vnmaniyar/.nvm/versions/node/v13.0.1/bin/node" "/Users/vnmaniyar/.nvm/versions/node/v13.0.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "--fallback-to-build" "--module=/Users/vnmaniyar/Documents/local-repos/hackathon-starter/node_modules/bcrypt/lib/binding/bcrypt_lib.node" "--module_name=bcrypt_lib" "--module_path=/Users/vnmaniyar/Documents/local-repos/hackathon-starter/node_modules/bcrypt/lib/binding" "--napi_version=5" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v79"
gyp ERR! cwd /Users/vnmaniyar/Documents/local-repos/hackathon-starter/node_modules/bcrypt
gyp ERR! node -v v13.0.1
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok
```

TLDR:
`npm install` fails because `node-pre-gyp` if forcing an https download and cannot find a local certificate.

Workaround:
Using bcryptjs instead of bcrypt helps but I read on other threads that it is slower (about ~30%) and has some minor limitations.

**Comment by YasharF** [comment]
https://travis-ci.org/sahat/hackathon-starter/jobs/610079345 using Node 13 worked just fine.  I suspect this is an issue with your setup.

**Comment by iamrudra** [comment]
Thanks @YasharF - I will check and report here. Thanks for this.

**Comment by iamrudra** [comment]
@YasharF - It seems like some proxy setting on my computer was breaking the install command. I connected to VPN and it worked fine. Thanks for the pointer - I was going crazy thinking about this.
